### PR TITLE
CLI to declare replicas temporary unavailable + Move rucio-admin replicas declare-bad to the new API : Closes #2277, #2671

### LINKED ISSUE #2277: Move rucio-admin replicas declare-bad to the new API
Motivation
----------
There is now a new API to declare bad PFNs much more efficient that the one currently used for `rucio-admin replicas declare-bad`. Will move to the new API.

### PR DESCRIPTION
**Pull request by cserf**
CLI to declare replicas temporary unavailable + Move rucio-admin replicas declare-bad to the new API : Closes #2277, #2671

### CODE REVIEW

File: bin/rucio-admin [hunk 1]
```diff
@@ -920,26 +922,61 @@ def declare_bad_file_replicas(args):
...
+
+    return SUCCESS
+
+
+@exception_handler
+def declare_temporary_unavailable_replicas(args):
+    """
+    %(prog)s show [options] <field1=value1 field2=value2 ...>
+
+    Declare a list of temporary unvailable replicas.
```
Comment (bari12) [inline bin/rucio-admin]: unavailable

**Review by bari12 — COMMENTED** [review]
For the line 933 with the print. In general we should follow english punctuation rules, so not put a space in front of `:`. 🤣

File: bin/rucio-admin [hunk 2]
```diff
@@ -920,26 +922,61 @@ def declare_bad_file_replicas(args):
...
+
+    """
+    client = get_client(args)
+    bad_files = []
+    if args.inputfile:
+        with open(args.inputfile) as infile:
+            for line in infile:
+                bad_file = line.rstrip('\n')
+                if bad_file.find('://') == -1:
```
Comment (mlassnig) [inline bin/rucio-admin]: `if '://' not in bad_file`

?

File: bin/rucio-admin (hunk 1 above)
Comment (cserf) [reply bin/rucio-admin]: Done

File: bin/rucio-admin (hunk 2 above)
Comment (cserf) [reply bin/rucio-admin]: Done

**Review by bari12 — APPROVED** [review]

**Pull request merged**